app/assets/javascripts/modules/inline-attachment-modal.js

```diff
@@ -36,6 +36,16 @@ InlineAttachmentModal.prototype.renderSuccess = function (result) {
   this.workflow.initComponents()
 }
 
+InlineAttachmentModal.prototype.insertAttachmentBlockSnippet = function (item) {
```

Comment:
It looks like inserting a snippet has become pretty standard among the modal modules, so I wonder if it's worth refactoring this into 'insertInline' and 'insertBlock' functions in a new 'ModalSnippet' utility - it doesn't really fit into 'ModalWorkflow', as we could have modals that don't insert snippets.

What do you think?